pkg/rancher/wait.go

```diff
@@ -152,6 +152,20 @@ func ToRestartRancherWebhookInstruction(k8sVersion string) (*applyinator.Instruc
 	}, nil
 }
 
+func ToCreateStvAggregationSecret(k8sVersion string) (*applyinator.Instruction, error) {
```

Comment:
Did we consider to put this secret in installer, instead of changing rancherd?

https://github.com/harvester/harvester-installer/blob/dd139f8bc7a12590edd84e64a4aada03e98487c1/pkg/config/templates/rancherd-10-harvester.yaml#L45

Reply:
Rancherd should be able to work independently, so it's better to fix the issue here.